src/components/People/PeopleTable.tsx

```diff
@@ -0,0 +1,249 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import classNames from 'classnames';
+import { useEffect, useState } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import { getPeople } from '../../api';
+import type { Person } from '../../types';
+import { TitleTableHeaders } from '../../types';
+import { getPreparedPeople } from '../../utils/getPreparedPeople';
+import { getSearchWith } from '../../utils/searchHelper';
+import { Loader } from '../Loader';
+import { PeopleFilters } from './PeopleFilters';
+import { PersonItem } from './PersonItem';
+import { SearchLink } from '../SearchLink';
+
+const DESCENDING_ORDER = 'desc';
+
+export const PeopleTable = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isMatchSearch, setIsMatchSearch] = useState(true);
+  const [isError, setIsError] = useState(false);
+
+  useEffect(() => {
+    setIsLoading(true);
+    getPeople()
+      .then(person => {
+        setPeople(person);
+      })
+      .catch(() => {
+        setIsError(true);
+        setTimeout(() => setIsError(false), 3000);
+      })
+      .finally(() => setIsLoading(false));
+  }, []);
+
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const sort = searchParams.get('sort') || null;
+  const order = searchParams.get('order') || null;
+
+  const sex = searchParams.get('sex') || null;
+  const query = searchParams.get('query') || '';
+  const centuries = searchParams.getAll('centuries') || [];
+
+  const showTable = !isLoading && !isError && isMatchSearch;
+  const showSideBar = !isLoading && !isError;
+
+  const setSearchWith = (params: any) => {
```

Comment:
you could use URLSearchParams (same as in `getSearchWith`). It's bad practice to write `any` without necessary